src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,52 @@
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+
+  &__item {
+    margin-left: 20px;
+
+    &--first {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    text-decoration: none;
+    color: $main_accent;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: normal;
+    text-transform: uppercase;
+    transition: color 0.3s;
+
+    &:hover {
+      color: $blue_accent;
+    }
+
+    &--first {
+      color: $blue_accent;
+    }
+
+    &--first::after {
```

Comment:
It's not `&--first` in this case, it should be `&--active` or just `.is-active`

Reply:
Not fixed